pages/deployment/aws.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Deploy Memgraph on AWS EC2 instance 
+description: Learn how to deploy your Memgraph instance on the AWS EC2 instance.
+---
+
+import { Card, Cards, Tabs, Tab } from 'nextra/components'
+
+# Deploy Memgraph in AWS EC2 instance
```

Comment:
in AWS instance or on AWS instance (I saw you used 'on' in other places)

Reply:
```suggestion
# Deploy Memgraph on AWS EC2 instance
```